definitions/variable/technology/electricity-operation.yaml

```diff
@@ -12,360 +12,82 @@
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Gas:
-    description: Operation Cost of Gas power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Gas|OCGT:
-    description: Operation Cost of Open Cycle Gas Turbine power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Gas|CCGT:
-    description: Operation Cost of Combined Cycle Gas Turbine power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Coal:
-    description: Operation Cost of coal power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Lignite:
-    description: Operation Cost of Lignite power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Oil:
-    description: Operation Cost of Oil power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Hydro|Reservoir:
-    description: Operation Cost of "simple Hydro reservoir " units
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Hydro|Pumped Storage:
-    description: Operation Cost of "Pumped Storage " units
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Hydro|Run of River:
-    description: Operation Cost of "Run of River" units
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Hydro|Ocean:
-    description: Operation Cost of Ocean Power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Solar:
-    description: Operation Cost of Photovoltaic power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Wind|OnShore:
-    description: Operation Cost of  Wind Onshore power plants
+    unit:
+    - MWh
+    - GWh
+
+- Operation Cost|Electricity|<Fuel>:
+    description: Operation Cost of <this fuel> power plants
     unit: EUR_2020
     skip-region-aggregation: true
-- Operation Cost|Electricity|Wind|OffShore:
-    description: Operation Cost of  Wind OffShore power plants
-    unit: EUR_2020
+
+- Reserve|Electricity|Automatic Frequency Restoration|<Fuel>:
+    description: Automatic Frequency Restoration Reserve available for <this fuel>
+     power plants; this amount is the maximum available for the grid operator considering
+     the commitment decisions taken by the model
+    unit:
+    - MWh
+    - GWh
     skip-region-aggregation: true
-- Operation Cost|Electricity|Electricity Storage:
-    description: Operation Cost of energy storage systems, excl. hydro storage
-    unit: EUR_2020
+
+- Reserve|Electricity|Frequency Containment|<Fuel>:
+    description: Frequency Containment Reserve available for <this fuel>
+     power plants; this amount is the maximum available for the grid operator considering
+     the commitment decisions taken by the model
+    unit:
+    - MWh
+    - GWh
```

Comment:
As above, why skip region aggregation on this variable?

Reply:
As above, please revert to `skip-region-aggregation: true`

Reply:
Got it.